README.md

```diff
@@ -40,7 +40,7 @@ Detailed documentation and concepts for each component can be found in the respe
 This repository contains a detailed developer documentation written in AsciiDoc. 
 The source files (AsciiDoc) are located [here](documentation/developer-guide) and are built using
 [Antora](https://antora.org/) which generates the documentation as HTML files. A site generated using Antora is self-contained and can be viewed entirely offline via a web browser without setting up a web server.
-To build the documentation locally, please check out and follow the instructions in [sds-documentation]( https://github.com/OpenManufacturingPlatform/sds-documentation).
+To build the documentation locally, please check out and follow the instructions in [sds-documentation](https://github.com/eclipse-esmf/eclipse-esmf.github.io).
```

Comment:
Please change the link text from `sds-documentation` to something else, e.g. `the documentation repository`. (I'd not use the repository name as a link text, since it could be misleading)